view/rest/handlers/routers.go

```diff
@@ -5,6 +5,8 @@ import "github.com/gorilla/mux"
 func (h *Handlers) ApiRouter() *mux.Router {
 	router := mux.NewRouter()
 
+	router.HandleFunc("/auth", h.Login).Methods("GET")
```

Comment:
did you test it works? do you use GET method with payload? Please read why GET is used for.